solutions/secure-regional-bucket/variables.tf

```diff
@@ -0,0 +1,202 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Token"
+  sensitive   = true
+}
+
+########################################################################################################################
+# KMS variables
+########################################################################################################################
+
+variable "kms_region" {
+  type        = string
+  default     = "us-south"
+  description = "The region in which KMS instance exists."
+}
+
+variable "existing_kms_guid" {
+  type        = string
+  default     = null
+  description = "The GUID of of the KMS instance used for the SCC COS bucket root Key. Only required if not supplying an existing KMS root key and if 'skip_cos_kms_auth_policy' is true."
+}
+
+variable "existing_kms_key_crn" {
+  type        = string
+  default     = null
+  description = "The CRN of an existing KMS key to be used to encrypt the SCC COS bucket. If not supplied, a new key ring and key will be created in the provided KMS instance."
+}
+
+variable "kms_endpoint_type" {
+  type        = string
+  description = "The type of endpoint to be used for commincating with the KMS instance. Allowed values are: 'public' or 'private' (default)"
+  default     = "private"
+  validation {
+    condition     = can(regex("public|private", var.kms_endpoint_type))
+    error_message = "The kms_endpoint_type value must be 'public' or 'private'."
+  }
+}
+
+variable "key_ring_name" {
+  type        = string
+  default     = "cross-region-key-ring"
+  description = "The name to give the Key Ring which will be created for the SCC COS bucket Key. Not used if supplying an existing Key."
+}
+
+variable "key_name" {
+  type        = string
+  default     = "cross-region-key"
+  description = "The name to give the Key which will be created for the SCC COS bucket. Not used if supplying an existing Key."
+}
```

Comment:
Same as other DA one.
- Need to remove `SCC`
- typo `commincating`
- remove this `skip_cos_kms_auth_policy` if we are not using.